lib/Service/src/Util.cpp

```diff
@@ -210,6 +211,17 @@ bool Util::isButtonTriggered(IButton& button, bool& lastState)
     return isTriggered;
 }
 
+int32_t Util::stepsPerSecondToMillimetersPerSecond(int16_t speedStepsPerSec)
+{
+    return (static_cast<int32_t>(speedStepsPerSec) * 1000 / static_cast<int32_t>(RobotConstants::ENCODER_STEPS_PER_M));
```

Comment:
These one or two line calculations can not be optimized by the compiler, because they are in the cpp file.
Because they will be called quite often, you have to put them to the header.

Reply:
Do I have to use `inline` in this case?

Reply:
if you define it inside the class body, no, otherwise yes. 
Example: https://github.com/BlueAndi/DroidControlShip/blob/fc781c6f861b32b8df4cb50f7d18e90ba03755af/lib/ArduinoNative/src/IPAddress.h#L124

Reply:
Thank you for the explanation! I have inlined them